src/components/Send/SendForm.tsx

```diff
@@ -214,13 +214,15 @@ export interface SendFormValues {
   txFee?: TxFee
   isCoinJoin: boolean
   numCollaborators?: number
+  selectedUtxos?: Array<Utxo>
```

Comment:
Is the property ` `SendFormValues#selectedUtxos` really need for the functionality of the feature?

Reply:
it was a left-over Thank you for pointing it out, btw.